index.bs

```diff
@@ -2336,14 +2340,16 @@ When this method is invoked, the [=authenticator=] must perform the following pr
     Return to the user agent:
         - |selectedCredential|'s [=credential ID=], if either a [=list=] of credentials of [=list/size=] 2 or greater was supplied
             by the client, or no such [=list=] was supplied. Otherwise, return only the below values.
-
-            Note: If the client supplies a [=list=] of exactly one credential and it was successfully employed, then its
-                [=credential ID=] is not returned since the client already knows it. This saves transmitting these bytes over
-                what may be a constrained connection in what is likely a common case.
-
         - |authenticatorData|
         - |signature|
-        - The [=user handle=] associated with |selectedCredential|.
+        - The [=user handle=] associated with |selectedCredential|, if no [=list=] of credentials was supplied by the client, or
+            no such [=list=] was supplied. Otherwise, do not return this value.
```

Comment:
I don't see any reason on why not. 

Its not a private information and RP may need it even if RP is supplying a set of credentialIDs as userHandle is helping RP for the lookup even in second factor case.